src/presence_analyzer/views.py

```diff
@@ -75,3 +81,32 @@ def presence_weekday_view(user_id):
 
     result.insert(0, ('Weekday', 'Presence (s)'))
     return result
+
+
+@app.route('/api/v1/presence_start_end/<int:user_id>', methods=['GET'])
+@jsonify
+def presence_start_end(user_id):
+    """
+    Calculate average time when user start the work and when user end the work.
+    """
+    data = get_data()
+    if user_id not in data:
+        log.debug('User %s not found!', user_id)
+        abort(404)
+
+    result = {}
+    for day in list(calendar.day_name):
+        result[day] = {'start': '0:0:0', 'end': '0:0:0'}
+
+    for item in data[user_id]:
+        result[calendar.day_name[item.weekday()]]['start'] = str(
+            timedelta(
+                seconds=seconds_since_midnight(
+                    data[user_id][item]['start']))
+        )
+        result[calendar.day_name[item.weekday()]]['end'] = str(
+            timedelta(
+                seconds=seconds_since_midnight(
+                    data[user_id][item]['end']))
+        )
```

Comment:
Also it would be a good idea to move all calculations to some util function to not to mess in the view - it should stay as clean as possible.